CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
 
 ### Added
+- Environment Variables for authentication: 
```

Comment:
I would just make this "Environment Variables for Basic Authentication
remove lines 11 & 12 (that's not new).
Did you add a line for `COURIER_BASE_URL`?